http/reactive-routes/pom.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>io.quarkus.ts.qe</groupId>
+        <artifactId>parent</artifactId>
+        <version>1.0.0-SNAPSHOT</version>
+        <relativePath>../..</relativePath>
+    </parent>
+    <artifactId>reactive-routes</artifactId>
+    <packaging>jar</packaging>
+    <name>Quarkus QE TS: reactive-routes</name>
```

Comment:
I think it's clearer to name it <name>Quarkus QE TS: HTTP: reactive-routes</name>.
In order to follow the same name convention with the rest of HTTP modules, what about to change the artifactId to http-reactive-routes too?